package.json

```diff
@@ -11,15 +11,18 @@
   "license": "ISC",
   "dependencies": {
     "bat-publisher": "https://github.com/brave-intl/bat-publisher.git#master",
-    "bignumber.js": "^9.1.0",
-    "bitcoinjs-lib": "^6.0.2",
-    "bs58check": "^2.1.2",
-    "country-list": "^2.2.0",
+    "bignumber.js": "^9.1.1",
+    "bitcoinjs-lib": "^6.1.0",
+    "bs58check": "^3.0.1",
+    "country-list": "^2.3.0",
     "currency-codes": "^2.1.0",
     "ethereum-address": "0.0.4",
-    "joi": "^17.6.0",
+    "joi": "^17.8.3",
     "lodash": "^4.17.21",
-    "superagent": "^8.0.0",
+    "superagent": "^8.0.9",
     "tiny-secp256k1": "^2.2.1"
+  },
+  "resolutions": {
```

Comment:
pretty sure this only works for yarn - did you mean overrides? https://docs.npmjs.com/cli/v9/configuring-npm/package-json#overrides

would be good to add an npm audit action here so it's obvious if the PR is still failing

Reply:
@tsmartt oops already done here - https://github.com/brave-intl/upload-settlements/pull/17